media/css/privacy/privacy-firefox-simple.scss

```
@@ -162,17 +162,50 @@ $border: 2px solid $color-marketing-gray-20;
 // * -------------------------------------------------------------------------- */
 // Tables
 
-.mzp-u-data-table  {
-    th {
-        vertical-align: bottom;
-    }
 
-    td {
-        border-color: $color-light-gray-30;
-        vertical-align: top;
-    }
+.mzp-u-data-table td {
+    border-color: $color-light-gray-30;
+    vertical-align: top;
 
     .mzp-u-list-styled {
-        margin-left: $spacing-md;
+        @include bidi((
+            (margin-left, $spacing-sm, margin-right, 0),
+        ));
+        margin-bottom: 0;
+    }
+}
+
+#lawful-bases + table td:nth-child(2){
+    min-width: 20ch;
+}
+
+@media (max-width: #{$content-xl - 1px}) {
+    .mzp-u-data-table {
+        thead {
+            display: none;
+        }
+
+        tr, th, td {
+            display: block;
+        }
+
+        td {
+            border: none;
+            padding: $spacing-sm 0;
+
+            &:first-child {
+                @include text-title-2xs;
+                font-weight: bold;
+            }
+        }
+
+        tr {
+        border-top: 1px solid $color-light-gray-50;
+        padding: $layout-2xs 0;
+
+        &:first-child {
+            border-top: none;
+        }
+        }
```

Thought linter dislikes whitespace issues but apparently not:

```suggestion
        tr {
            border-top: 1px solid $color-light-gray-50;
            padding: $layout-2xs 0;

            &:first-child {
                border-top: none;
            }
        }
```

(Noticed this also in the original stylesheet BTW.)